actors/builtin/market/policy.go

```diff
@@ -5,15 +5,16 @@ import (
 	"github.com/filecoin-project/go-state-types/big"
 
 	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v2/actors/runtime"
 )
 
 // The number of epochs between payment and other state processing for deals.
 const DealUpdatesInterval = builtin.EpochsInDay // PARAM_SPEC
 
 // The percentage of normalized cirulating
 // supply that must be covered by provider collateral in a deal
-var ProviderCollateralSupplyTarget = builtin.BigFrac{
-	Numerator:   big.NewInt(5), // PARAM_SPEC
+var ProviderCollateralSupplyTargetV1 = builtin.BigFrac{
```

Comment:
I chose to keep naming these after the network version at which they came into effect, but we don't need to do this.

Reply:
> We could introduce version suffixes only for inline updates and insist that versioned constants become unversioned when we remove inline code at the next release update.

👍

Reply:
Ok